shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -234,6 +239,7 @@ - (void)loadView {
 
 - (void)viewDidLoad {
   [self configureTrackingArea];
+  [self listenForMetaModifiedKeyUpEvents];
```

Comment:
Let's put this in `viewWillAppear` for parity. (IIRC it doesn't matter on macOS the way it would on iOS, but it's conceptually cleaner.)

Reply:
Done